Documentation/gh-pages/source/compiling.rst

```diff
@@ -126,17 +126,20 @@ On a RedHat/Fedora Linux system, the packages can be installed through the comma
 Required dependencies
 ---------------------
 
-- CMake_ version 3.13 or higher
-- g++ version 6.1 or higher (if compiling with GCC_)
-- clang++ version 3.4 or higher (if compiling with Clang_)
+- CMake_ version |CMakeVersion| or higher
+- g++ version |gccVersion| or higher (if compiling with GCC_)
+- clang++ version |clangVersion| or higher (if compiling with Clang_)
 - GSL_
-- ROOT_ version 6.16 or higher
+- Boost_ version |BoostVersion| or higher
+- ROOT_ version |ROOTVersion| or higher
+
+ - \--enable-minuit2 (if you want to use KaFit)
+ - \--enable-fftw3 (if you want to use KEMField)
```

Comment:
```suggestion
- ROOT_ version |ROOTVersion| or higher (built with flag ``--enable-fftw3``)
```

One can't really use `Kassiopeia` without `KEMField`. Also `KaFit` is not part of Kassiopeia.